website/docs/reference/resource-properties/constraints.md

```diff
@@ -518,3 +518,71 @@ alter table schema_name.my_model add constraint 472394792387497234 check (id > 0
 </div>
 
 </WHCode>
+
+### Custom constraints on models for advanced configuration of tables
+
+In dbt Cloud, you can use custom constraints on models for the advanced configuration of tables. Different data warehouses support different syntax and capabilities. However, custom constraints allow you to add configuration to specific columns.
```

Comment:
just double check if this is something only for dbt cloud? the github issues i think is for core users so wanted to confirm this was what we wanted to communicate @b-per ? https://github.com/dbt-labs/dbt-adapters/issues/85#issuecomment-2310772737